launcher.py

```diff
@@ -65,10 +137,63 @@ def deploy_contract(self) -> bool:
         )
         return False
 
+    def clean_config_entry(self, line: str) -> str:
+        ''' Clean the line that forms the entry for the config.exs
+        '''
+        line = re.sub(r'"', '', line)
+        line = re.sub(r',', '', line)
+        return line
+
+    def update_contract_exchanger(self):
+        ''' Update the contract exchanger service with the details of the
+        deployed contract
+        '''
+        contract_details = {}
+        with open(os.path.expanduser('~') + '/config.exs') as contract:
+            contract_list = [x.strip('\n') for x in contract.readlines()]
+            for entry in contract_list:
+                if ('Mix.Config' in entry or 'omg_eth' in entry):
+                    continue
+                if 'contract_addr' in entry:
+                    contract_details['contract_addr'] = \
+                        self.clean_config_entry(entry.split(' ')[3])
+                if 'txhash_contract' in entry:
+                    contract_details['txhash_contract'] = \
+                        self.clean_config_entry(entry.split(' ')[3])
+                if 'authority_addr' in entry:
+                    contract_details['authority_addr'] = \
+                        self.clean_config_entry(entry.split(' ')[3])
+
+        requests.post(
```

Comment:
This manual parsing of config files is brutal. What do you think about using .toml as a format for `elixir-omg` config files?

Reply:
I agree. Any structured data format would be better than parsing a `config.exs`. Something for after public testnet?